cmd/zoekt-archive-index/main.go

```diff
@@ -220,12 +221,24 @@ func main() {
 	}
 	archive := flag.Args()[0]
 
+	largeFilesMap := map[string]struct{}{}
```

Comment:
Could this be encapsulated in a method on the Options struct (similar to [`shardName`](https://sourcegraph.com/github.com/sourcegraph/zoekt@395f561fba1e03ec7c8d0615a0b0b89db2f0afed/-/blob/build/builder.go#L150:19))? Method name would be something like `largeFilesSet`.